roles/setup_gitea/tasks/install.yml

```diff
@@ -0,0 +1,337 @@
+
+          [session]
+          PROVIDER_CONFIG = /var/lib/gitea/data/sessions
+          PROVIDER = file
+
+          [picture]
+          AVATAR_UPLOAD_PATH = /var/lib/gitea/data/avatars
+          REPOSITORY_AVATAR_UPLOAD_PATH = /var/lib/gitea/data/repo-avatars
+
+          [attachment]
+          PATH = /var/lib/gitea/data/attachments
+
+          [log]
+          ROOT_PATH = /var/lib/gitea/data/log
+          MODE = console
+          LEVEL = info
+
+          [security]
+          INSTALL_LOCK = true
+          SECRET_KEY = {{ _sg_secret_key.stdout_lines[-1] }}
+          REVERSE_PROXY_LIMIT = 1
+          REVERSE_PROXY_TRUSTED_PROXIES = *
+          INTERNAL_TOKEN = {{ _sg_internal_token.stdout_lines[-1] }}
+          PASSWORD_HASH_ALGO = pbkdf2
+
+          [service]
+          DISABLE_REGISTRATION = false
+          REQUIRE_SIGNIN_VIEW = false
+          REGISTER_EMAIL_CONFIRM = false
+          ENABLE_NOTIFY_MAIL = false
+          ALLOW_ONLY_EXTERNAL_REGISTRATION = false
+          ENABLE_CAPTCHA = false
+          DEFAULT_KEEP_EMAIL_PRIVATE = false
+          DEFAULT_ALLOW_CREATE_ORGANIZATION = true
+          DEFAULT_ENABLE_TIMETRACKING = true
+          NO_REPLY_ADDRESS = noreply.localhost
+
+          [lfs]
+          PATH = /var/lib/gitea/git/lfs
+
+          [mailer]
+          ENABLED = false
+
+          [openid]
+          ENABLE_OPENID_SIGNIN = true
+          ENABLE_OPENID_SIGNUP = true
+
+          [cron.update_checker]
+          ENABLED = false
+
+          [repository.pull-request]
+          DEFAULT_MERGE_STYLE = merge
+
+          [repository.signing]
+          DEFAULT_TRUST_MODEL = committer
+
+          [oauth2]
+          JWT_SECRET = {{ _sg_jwt_secret.stdout_lines[-1] }}
+
+- name: Create the Gitea deployment
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: gitea
+        namespace: "{{ sg_namespace }}"
+      spec:
+        selector:
+          matchLabels:
+            app: gitea
+        strategy:
+          type: Recreate
+        template:
+          metadata:
+            labels:
+              app: gitea
+          spec:
+            volumes:
+              - name: gitea-app-ini
+                configMap:
+                  name: gitea-app-ini
+            containers:
+              - name: gitea
+                volumeMounts:
+                  - name: gitea-app-ini
+                    subPath: app.ini
+                    mountPath: /etc/gitea/app.ini
+                image: "{{ sg_gitea_image }}"
+                ports:
+                  - containerPort: 3000
+                livenessProbe:
+                  httpGet:
+                    path: /api/healthz
+                    port: http
+                  initialDelaySeconds: 200
+                  timeoutSeconds: 5
+                  periodSeconds: 10
+                  successThreshold: 1
+                  failureThreshold: 10
+            serviceAccountName: anyuid
```

Comment:
no_log for tasks with sensitive information. In some roles we started adding a flag like `sg_no_log: true` (default) in case debug is needed.